src/app/website/posts/zkp.markdown

```diff
@@ -0,0 +1,74 @@
+---
+title: Zero-knowledge Proofs&#58; An Intuitive Explanation
+date: 2019-05-13
+author: Vanishree Rao
+---
+
+If you are hearing about zero-knowledge proofs -- a fascinating cryptography tool, for the first time, you are likely to say "Hah! That sounds impossible." Read on to get an intuitive understanding of what they are. But first, some background. Zero-knowledge proofs (or ZKPs)  were [invented](http://people.csail.mit.edu/silvio/Selected%20Scientific%20Papers/Proof%20Systems/The_Knowledge_Complexity_Of_Interactive_Proof_Systems.pdf) by Shafi Goldwasser, Silvio Micali, and Charles Rackoff in 1985. Ever since, ZKPs have been one of the most active areas of research in Cryptography. Moreover, recently, they are enjoying significant impact on real-world applications, specifically on blockchain technologies. [Zcash](https://z.cash/technology/), a pioneering blockchain project, employed ZKPs to achieve anonymity in financial transactions. At O(1)Labs, we are building CODA, the first succinct blockchain, using ZKPs. No matter how many transactions are recorded on the blockchain, the blockchain remains at most the size of a few tweets. 
+
+
+
+## What are Zero-knowledge Proofs?
+
+ZKPs are a powerful tool that enable you do the following. Let’s say you have a secret. Using ZKPs, you can prove that you have the secret without even revealing the it!  
```

Comment:
maybe if you reword this a bit it could be a good introductory sentence.

Reply:
It could be:
`ZKPs are a powerful tool that allow you to prove that you have a secret, without revealing it to anyone".

Reply:
Done.